### DISCUSSION TITLE
Why can't my OsgEarth project display normally after overlaying the .ive file?

### DISCUSSION THREAD

**Discussion by MRdesperado1** [pelicanmapping/osgearth#1809] [Support Q&A] [answered]
![image](https://user-images.githubusercontent.com/38163799/134951300-664a6d51-81c8-476f-aa35-f2f45138b24d.png)
The .ive file seen through osgviewer is like this.
But when I superimposed it on my project, it became like this.
![1](https://user-images.githubusercontent.com/38163799/134952111-9d7346a4-4727-4596-acef-3c4732269142.png)
![image](https://user-images.githubusercontent.com/38163799/134952201-dfb359b5-d9f1-4f02-947e-fb21e669cf4c.png)
Most of the .ive files are not displayed and show this thin strip. I don't know why.
My overlay code is as follows:
world = osgDB::readNodeFile("dizhoujie.ive");
	osg::ref_ptr<osgEarth::DrapeableNode> dnode = new  osgEarth::DrapeableNode();
	dnode->addChild(world);
	mapNode->addChild(dnode);
I use OSG version 3.6.3 and osgearth version 2.10.

**Comment by gwaldron** [comment, marked as answer]
What coord system is your model in? It needs to be geocentric (ECEF) coordinates for this to have a chance of working.

**Reply by MRdesperado1** [reply]
Thank you very much. The problem has been solved.
